bool shakeHand_over
int32 shakeHand_count

